src/super_gradients/examples/predict/segmentation_predict.py

```diff
@@ -0,0 +1,16 @@
+from super_gradients.common.object_names import Models
+from super_gradients.training import models
+
+# Note that currently only XXX, YYY and ZZZ are supported.
+# model = models.get(Models.PP_LITE_B_SEG, pretrained_weights="cityscapes")
+model = models.get(model_name=Models.PP_LITE_T_SEG75, arch_params={"use_aux_heads": False}, num_classes=19, pretrained_weights="cityscapes")
+
+IMAGES = [
+    "../../../../documentation/source/images/examples/countryside.jpg",
+    "../../../../documentation/source/images/examples/street_busy.jpg",
+    "https://cdn-attachments.timesofmalta.com/cc1eceadde40d2940bc5dd20692901371622153217-1301777007-4d978a6f-620x348.jpg",
```

Comment:
Let's not use URLs form the domains we do not control. @Louis-Dupont I recall a similar issue with some of the other PRs and if I'm not mistaken Ran Rubin uploaded some image to our S3 for this reason. Can you recall where it were and what was the url?

Reply:
this is the same as all "predict" examples
if changes are made it should be done to all prediction examples

Reply:
The fact that we have something wrong (Wrong url for detection predict) does not mean we should introduce new wrong things.
Please change `https://cdn-attachments.timesofmalta.com/cc1eceadde40d2940bc5dd20692901371622153217-1301777007-4d978a6f-620x348.jpg` to `https://deci-pretrained-models.s3.amazonaws.com/sample_images/beatles-abbeyroad.jpg`